README.md

```diff
@@ -42,23 +44,31 @@ has been destroyed.
 - [4. Saving More Information About Versions](#4-saving-more-information-about-versions)
   - [4.a. Finding Out Who Was Responsible For A Change](#4a-finding-out-who-was-responsible-for-a-change)
   - [4.b. Associations](#4b-associations)
-    - [4.b.1. Known Issues](#4b1-known-issues)
```

Comment:
4.b.1. disappears, because I defined `--max-depth 3`, with a higher max-depth, even more items would appear, 4.b.1. is the anomaly here, being the only ToC entry that deep.